aii-core/src/main/eg/aii-shellfe.conf

```diff
@@ -15,3 +15,28 @@
 # The default is profile_ for backward compatibility
 # Comment this out when not prefix is needed
 profile_prefix = profile_
+
+# Directory on TFTP server where PXELINUX configuration files are.
+# Must be an absolute path or none to disable PXELINUX configuration.
```

Comment:
set quotes around none, as you expect the literal value `npbdir = none` here, right? (otherwsie, people might think to just leave it empty)